Add update service node page for docker in Lisk Service v0.7.0 (#1850)
LiskHQ/lisk-service: docs/antora/modules/ROOT/pages/management/docker.adoc

:url_config: configuration/docker.adoc
:url_FLUSHALL: https://redis.io/commands/FLUSHALL

> NOTE: To upgrade Lisk Service to a desired version, please check out the specific version with *`git checkout <tag>`*, and then follow the existing process described below.

Reviewer: what process are you referring to here exactly?

Author: Yes this is a good point Mona (The same as in the source service guide), so I have reworded it accordingly.

== Build Docker image

.Inside the lisk-service root folder
----
make build
----
This creates the necessary Docker images to start Lisk Service in the containers.

== Start

.Inside the lisk-service root folder
----

> cp ./docker/example.env .env

Reviewer: i don't think this needs t be done every time Lisk Service is started, so it doesn't really fit here.

Reviewer: but this is not about starting it for the first time, it is about how to start it in general. I don't think you need to do it every time you start Lisk Service. So it shouldn't be mentioned here

Reviewer: As mentioned, this is not about starting for the first time. This step is already included at the right location, which is the setup guides: https://liskhq.github.io/lisk-docs/lisk-service/0.7/setup/docker.html#connecting-lisk-service-to-a-blockchain-node

:url_snapshot_config: configuration/index.adoc
:url_backups: management/snapshots.adoc

> NOTE: To upgrade Lisk Service to a desired version, please check out the specific version with *`git checkout <tag>`*, and then create the Docker image, followed by starting Lisk Service as shown in the commands below.

Reviewer: How to update Lisk Service should not be a note, it should at least be a separate section, and every step necessary to update Lisk Service should be explained it that very section, and supported, if possible, by code snippets the user can simply copy.

== Logging

The possibility exists here to either use the `docker` commands or the `make` commands to check the logs, and both options are listed here below.

For `docker`, to check the logs for the different microservices of Lisk Service, use the command `docker container logs CONTAINER`, where `CONTAINER` is the respective Docker container that holds the logs you wish to view.

For example, to see the logs for the Gateway microservice, execute the following command:

----
docker container logs lisk-service_gateway_1
----

To check the logs for different microservices using the `make` commands, the following commands listed below can be used:

> ----
> make logs
> ----
> The above command, displays the logs for all the microservices that have been logged so far, and that are currently existing.
> ----
> make logs-live
> ----
> The above command displays the logs for all the microservices currently logged so far, and also continues to display the latest logs.
> ----
> make logs-blockchain-connector
> ----
> The above command is similar to the `make logs` command, but only displays the appropriate logs for the specified microservice.
> For example, in this case only for the `blockchain-connector`.
> ----
> make logs-live-blockchain-connector
> ----
> Finally, the above command is similar to the `make logs-live` command, but only displays the logs for the specified microservice.

Reviewer: Maybe add the descriptions as label of the snippet, to make it visually clearer which descrition belongs to which command.

```suggestion
.Displays the logs for all the microservices that have been logged so far, and that are currently existing
----
make logs
----

.Displays the latest logs for all the microservices
----
make logs-live
----

.Displays the logs for the specified microservice.
----
make logs-blockchain-connector
----
In the above example, it will display logs only for the `blockchain-connector` microservice.

.Displays the latest logs for the specified microservice.
----
make logs-live-blockchain-connector
----
In the above example, it will display latest logs only for the `blockchain-connector` microservice.

```

> For example, in this case also only for blockchain-connector here.

Reviewer: ```suggestion
```

----
docker container logs lisk-service_gateway_1
----

To check the logs for different microservices using the `make` commands, the following commands listed below can be used:

.Displays the logs for all the microservices that have been logged so far, and that are currently existing
----
make logs
----

.Displays the latest logs for all the microservices
----
make logs-live
----

.Displays the logs for the specified microservice.
----
make logs-blockchain-connector
----
In the above example, it will display logs only for the `blockchain-connector` microservice.

.Displays the latest logs for the specified microservice.
----
make logs-live-blockchain-connector
----
In the above example, it will display latest logs only for the `blockchain-connector` microservice.

Replace the `blockchain-connector` with the specific service name required.

== Upgrade Lisk Service

To upgrade the Docker container to a desired version, please follow one of the two options below. To find all the tagged versions for Lisk Service, please check the available tags (https://github.com/LiskHQ/lisk-service/tags) on GitHub.

> ==== Option A - Download pre-built images from DockerHub

Reviewer: Option A should be the recommended way to upgrade Lisk Service, imho Option A and B should be switched, because B looks easier

Reviewer: Also, imho it would be nice to use tabs to switch between Option A & B

Author: OK, I have now added tabs.

Author: OK

> . Update `docker-compose.yml` to download the specific image versions from the DockerHub as shown below. Update the `0.7.0` tag against the specified images with the desired version.

Reviewer: ```suggestion
. Update `docker-compose.yml` to download the specific image versions from the DockerHub as shown below. 
Update specified images with the desired version, for example `0.7.0`.
```

+

> ----

Reviewer: ```suggestion
.docker-compose.yml
[source,yaml]
----
```

blockchain-app-registry:
    image: lisk/service_blockchain_app_registry:0.7.0
    ...

  blockchain-connector:
    image: lisk/service_blockchain_connector:0.7.0
    ...

  blockchain-indexer:
    image: lisk/service_blockchain_indexer:0.7.0
    ...

  blockchain-coordinator:
    image: lisk/service_blockchain_coordinator:0.7.0
    ...

  transaction-statistics:
    image: lisk/service_transaction_statistics:0.7.0
    ...

  fee-estimator:
    image: lisk/service_fee_estimator:0.7.0
    ...

  gateway:
    image: lisk/service_gateway:0.7.0
    ...

  market:
    image: lisk/service_market:0.7.0
    ...

  export:
    image: lisk/service_export:0.7.0

----

. Stop Lisk Service
+
----
make stop
----
NOTE: In case the database needs to be flushed please execute the `make down` command instead. Please check the {url_rel_notes}[GitHub release notes] for the relevant version to verify if the existing databases need to be flushed.

. Start Lisk Service in the containers.
+
----
make up
----

==== Option B - Build images locally

> Build the images locally using the following steps.

Reviewer: At what point does it describe how to download and install the new version?

Author: Done as suggested 👍
